packages/icons/src/IconBase.tsx

```diff
@@ -91,11 +59,13 @@ export interface IconBaseProps
    * @example ["brand", "inherit"]
    */
   color?: HvColorAny | HvColorAny[];
-  /** Sets one of the standard sizes of the icons */
+  /** The size of the SVG icon */
```

Comment:
It seems like an unexpected behavior that using the prop size can change the color, maybe we could add something a note to the prop description?

Reply:
good point - added